code/modules/uplink/uplink_items.dm

```diff
@@ -1902,6 +1902,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 20
 	restricted_roles = list("Chaplain")
 	surplus = 5 //Very low chance to get it in a surplus crate even without being the chaplain
+	
+/datum/uplink_item/role_restricted/horror
+	name = "Horror-in-a-box"
+	desc = "When dissecting a head of a dead Nanotrasen scientist, our surgeons noticed incredibly peculiar creature inside and menaged to extract it into safe containment. \
+	Either a failed experiment or otherworldy monster, this creature has been trained to aid whoever wakes it up. If you aren't afraid of it entering your head, it can prove an useful ally. \
+	We take no responsibility for your newfound madness and accept no refunds."
```

Comment:
```suggestion
	desc = "When dissecting the head of a dead Nanotrasen scientist, our surgeons noticed an incredibly peculiar creature inside and managed to extract it into safe containment. \
	Either a failed experiment or otherworldly monster, this creature has been trained to aid whoever wakes it up. If you aren't afraid of it entering your head, it can prove a useful ally. \
	We take no responsibility for your newfound madness and accept no refunds."
```